R/animint.R

```diff
@@ -568,12 +568,14 @@ saveLayer <- function(l, d, meta){
           chunk.vec <- g.data[[chunk.var]]
           counts <- table(chunk.vec)
           if(length(counts) == 1){
-            stop("only 1 chunk") # do we ever get here?
-          }
-          if(all(counts == 1)){
-            FALSE #each chunk has only 1 row -- chunks are too small.
-          }else{
+            ##stop("only 1 chunk") # do we ever get here?
             TRUE
```

Comment:
this `stop` is not necessary (I added it while debugging the initial implementation of the compiler code which divides each data set into chunks). It was never being run in the past because most the data sets of most showSelected layers have more than one value of the showSelected variable. However I am currently developing such a data viz, so we need to get rid of this `stop`.